R/chromote_session.R

```diff
@@ -569,16 +589,15 @@ ChromoteSession <- R6Class(
   private = list(
     session_id = NULL,
     target_id = NULL,
-    is_active_ = NULL,
+    session_is_active = NULL,
+    target_is_active = NULL,
```

Comment:
What do you think about `target_is_alive`? (I'm asking honestly, not suggesting that it should or should not be that name)

Reply:
I don't mind changing the name, but calling it `alive` does introduce another inconsistency — `Chromote$is_alive()` is always accurate because we have a connection to the process that we can check; but there's currently no analog for the target. I could add an `is_alive()` method that checks if the target ID is found in the chromote instance, but I'm not sure if that's worth it (plus calling that repeatedly might be expensive).

So while I agree that "active" isn't the best name, and has some potential for confusion, I think changing it to "alive" is a net neutral because it just introduces a different source confusion.

Reply:
OK, let's keep it as "active" then.